### DISCUSSION TITLE
binding.StringList, Remove / RemoveAll missing?

### DISCUSSION THREAD

**Discussion by zarkopopovski** [fyne-io/fyne#3298] [Q&A] [answered]
Hi,

i am trying to remove a element from the List initialized with StringList data structure, but StringList doesnt implement Remove / RemoveAll methods, how can i remove a element or update the List with the different data?
Thanks

**Comment by Jacalz** [comment, marked as answer]
This functionality is currently missing. There is an open issue tracking this support: https://github.com/fyne-io/fyne/issues/3100

**Reply by Jacalz** [reply]
A potential workaround is to get the slice using `.Get()`, then modify it and paste it into `.Set()`. It is not a very clean solution but should work.

It is worth noting that removing single elements from slices is slow given that all elements after need to be shifted. Be careful with removing single items unless you really need to.
